lib/resources/r4/coverage.yaml

```diff
@@ -0,0 +1,230 @@
+---
+name: Coverage
+field_name_base_url: http://hl7.org/fhir/r4/coverage-definitions.html#Coverage
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  action: create
+  description: The type of the FHIR resource.
+  example: |
+    {
+      "resourceType": "Coverage"
+    }
+  note: resourceType must be Coverage
+  url: http://hl7.org/fhir/r4/json.html#resources
+
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  action: create
+  description: The status of the coverage.
+  example: |
+    {
+      "status": "active"
+    }
+  note: Currently only supports `active`.
+
+- name: subscriber
+  required: 'No'
+  cardinality: 0..1
+  type: Reference (Patient)
+  action: create
+  description: The subscriber to the policy.
+  example: |
+    {
+      "subscriber": {
+        "reference": "Patient/1626016",
+      }
+    }
+  note: Currently subscriber only supports Patient reference.
+
+- name: subscriberId
+  required: 'No'
+  cardinality: 0..1
+  type: string
+  action: create
+  description: ID assigned to the subscriber.
+  example: |
+    {
+      "subscriberId": "1626016"
+    }
+
+- name: beneficiary
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference (Patient)
+  action: create
+  description: Plan beneficiary.
+  example: |
+    {
+      "beneficiary": {
+        "reference": "Patient/1626016"
+      }
+    }
+
+- name: dependent
+  required: 'No'
+  cardinality: 0..1
+  type: string
+  action: create
+  description: Dependent number.
+  example: |
+    {
+      "dependent": "1626016"
+    }
+
+- name: relationship
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  action: create
+  description: Indicates the beneficiary relationship to the subscriber.
+  example: |
+    {
+      "relationship": [
+        {
+          "coding": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/subscriber-relationship",
+              "code": "child"
+            }
+          ]
+        }
+      ]
+    }
+  binding:
+    description: Indicates the beneficiary relationship to the subscriber.
+    terminology:
+      - display: Coverage Relationship
```

Comment:
need new display and system

Reply:
https://github.com/cerner/fhir.cerner.com/pull/304/commits/f9da3987687b66469c6ccfe041d4c8f7b0530d16#diff-af9c966415db83d073f1bcec5ed6f875R101-R102